Movio/app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,7 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorPrimaryAccent</item>
+        <item name="android:windowBackground">@color/colorPrimaryAccent</item>
```

Comment:
toto nie je uplne riesenie.
upravenie tohto atributu moze sposobit zmeny farby roznych komponent, ktore ju pouzivaju.
kedze pre kazdu temu si definujeme colorAccent, chceme pouzit tento atribut dynamicky. to sa robi tak ze pre View, ktoreho pozadie chceme menit dynamicky, nastavime android:background na ?colorAccent. tak sa aplikuje vzdy farba definovana v atribute colorAccent pre prave zvolenu temu.